vercelへのデプロイエラー

## 実装内容
vercelへデプロイ時に発生したビルドエラー
```
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: @typescript-eslint/eslint-plugin@7.17.0
npm error Found: @typescript-eslint/parser@5.62.0
npm error node_modules/@typescript-eslint/parser
npm error   dev @typescript-eslint/parser@"^5.62.0" from the root project
npm error   @typescript-eslint/parser@"^5.4.2 || ^6.0.0 || 7.0.0 - 7.2.0" from eslint-config-next@14.2.5
npm error   node_modules/eslint-config-next
npm error     dev eslint-config-next@"^14.2.5" from the root project
npm error
npm error Could not resolve dependency:
npm error peer @typescript-eslint/parser@"^7.0.0" from @typescript-eslint/eslint-plugin@7.17.0
npm error node_modules/@typescript-eslint/eslint-plugin
npm error   dev @typescript-eslint/eslint-plugin@"^7.17.0" from the root project
```

--- a/package.json
+++ b/package.json
@@ -43,7 +43,7 @@
     "@types/react": "^18",
     "@types/react-dom": "^18",
     "@typescript-eslint/eslint-plugin": "^7.17.0",
-    "@typescript-eslint/parser": "^5.62.0",
+    "@typescript-eslint/parser": "^7.17.0",
     "eslint": "^8.57.0",
     "eslint-config-next": "^14.2.5",
     "eslint-config-prettier": "^9.1.0",

